File: week1/sum.c
#include<stdio.h>
#include<stdlib.h>
int main()
{
    int n1;
    float n2,sum;
    printf("enter two number\n");
    scanf("%d%f",&n1,&n2);
    sum=n1+n2;
    printf("the sum=%d+%f= %f \n",n1,n2,sum);
}

File: java/SumOf2/src/Sum.java
import java.util.Scanner;
public class Sum {
public static void main(String ar[]) {
	Scanner s=new Scanner(System.in);
	System.out.println("enter the number");
	int num=s.nextInt();
	int sum=0;
	for(int i=1;i<num;i++)
	{
		sum=sum+i;
	}
	System.out.println("\n"+sum);
}
}
